server/api.js

```diff
@@ -733,12 +736,13 @@ router.post("/bid", async (req, res) => {
 				return res.status(400).json({ code: "DUPLICATE_ENTRY" });
 			}
 			const bidQuery = `
-                      INSERT INTO bid (tender_id, bidder_id, bidding_date, status, bidding_amount, cover_letter, suggested_duration_days)
-					  VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING bid_id
+                      INSERT INTO bid (tender_id, bidder_id, buyer_id, bidding_date, status, bidding_amount, cover_letter, suggested_duration_days)
+					  VALUES ($1, $2, $3, $4, $5, $6, $7, $8) RETURNING bid_id
```

Comment:
This shouldn't be necessary. I just noticed about this, but the bid table shouldn't have a `buyer_id` field. We know the buyer because we know the `tender_id`.